src/content/developers/docs/gas/index.md

```diff
@@ -36,11 +36,50 @@ Although a transaction includes a limit, any gas not used in a transaction is re
 ![Diagram showing how unused gas is refunded](../transactions/gas-tx.png)
 _Diagram adapted from [Ethereum EVM illustrated](https://takenobu-hs.github.io/downloads/ethereum_evm_illustrated.pdf)_
 
+## Why can gas fees get so high? {#Why-can-gas-fees-get-so-high}
+
+High gas fees are due to the popularity of Ethereum. Performing any operation on Ethereum requires consuming gas. This includes calculations, storing or manipulating data, or transferring tokens, each consuming different amounts of "gas" units. As dapp functionality grows more complex, the number of operations a smart contract performs grows too, and all contribute to high amounts of gas that must be paid for.
+
+Gas is paid for with small amounts of ether (denominated in gwei, or (10<sup>-9</sup> ETH)). This value represents the amount of ether being paid per "unit of gas", not for the entire transaction. Since gas is capped within a block, under the current system, this means that users must out-bid one another to be included in the next block if demand exceeds capacity. Gas cost alone does not actually determine how much we have to pay for a particular transaction. To calculate the transaction fee we have to multiply the gas cost by gas price, which is measured in gwei.
+
+This video about gas fees explains fully why fees are so expensive:
+
+https://www.youtube.com/embed/Yh8cHUB-KoU
+
+## Initiatives to reduce gas costs {#Initiatives-to-reduce-gas-costs}
+
+With the new network upgrades of Ethereum 2.0 (also known as Eth2 or Serenity). This should ultimately address some of the gas fee issues, which will in turn enable the platform to process thousands of transactions per second and scale globally.
+
+Layer 2 scaling is a primary initiative to greatly improve gas, user experience and scalability, our page on layer 2 scaling and rollups can be found here:
+-[Layer 2 scaling](/developers/docs/layer-2-scaling/)
```

Comment:
@DevAranCarter this URL just got updated 😂  sorry!...  Also, in style with much of the site, maybe condense to 
`Layer 2 scaling is a primary initiative to greatly improve gas costs, user experience and scalability. [More on layer 2 scaling](/developers/docs/scaling/layer-2-rollups/).`

Then let's get this merged =)

Reply:
All changed, thanks wackerow!